go/common/ledger/ledger.go

```diff
@@ -21,7 +22,7 @@ const (
 
 var (
 	// ListingDerivationPath is the path used to list and connect to devices by address.
-	ListingDerivationPath = []uint32{PathPurpose, ListingPathCoinType, ListingPathAccount, ListingPathChange, ListingPathIndex}
+	ListingDerivationPath = []uint32{PathPurposeEntity, ListingPathCoinType, ListingPathAccount, ListingPathChange, ListingPathIndex}
```

Comment:
Since this hasn't changed, it means `oasis-node` will still list connected Ledger devices using the `<mnemonic> / 44' / 474' / 0' / 0 / 0` [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki) path (reference: #3053).

Reply:
Yes, in order to also list Validator devices `ListDevices` should take into account a `ListingDerivationPath` with `PathPurposeConsensus`